src/scss/_sofie-rundown.scss

```diff
@@ -0,0 +1,13 @@
+.transparent-red-zebra-opacity {
+  &:before {
+    content: '';
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    background: var(--transparent-and-red-stripes);
+    z-index: 10;
+    opacity: 0.8;
```

Comment:
Why is the opacity added?

Reply:
Stripes happen to make media title unreadable.